Wikipedia/Code/ReadingListsViewController.swift

```diff
@@ -294,6 +294,7 @@ class ReadingListsViewController: ColumnarCollectionViewController, EditableColl
 
     override func apply(theme: Theme) {
         super.apply(theme: theme)
+        view.backgroundColor = theme.colors.paperBackground
```

Comment:
this is for ->
![screen shot 2018-07-25 at 9 02 39 am](https://user-images.githubusercontent.com/9299317/43202651-00f3b786-8fea-11e8-8bf7-75bcf66a0287.png)